### PR TITLE
Improve associations section in Readme

### PR DESCRIPTION
**Pull request by westonganger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -840,17 +840,15 @@ string, please try the [paper_trail-globalid][37] gem.
...
-`paper_trail-association_tracking` and keep running the existing tests related
-to association tracking. This arrangement will be maintained for a few years, if
-practical.
+If you would like to track and reify associations you can utilize the following PT plugin: [paper_trail-association_tracking][6]
+
+A little history lesson, discussed as early as 2009, and first implemented in late 2014, association
+tracking was part of PT core until 2018 as an experimental feature and was use at your own risk. The gem now
+maintains a list of known issues and we hope the community can help remove some of them via PR's.
+
+In PaperTrail v10, the association tracking feature was extracted to a gem to simplify things in PaperTrail and
```
Comment (jaredbeck) [inline README.md]: Since it's not a breaking change, I was thinking this could happen in 9.2.

Comment (westonganger) [reply README.md]: Fair enough lets go with that then

File: README.md [hunk 2]
```diff
@@ -840,17 +840,15 @@ string, please try the [paper_trail-globalid][37] gem.
...
-
-To avoid breaking changes, `paper_trail` will have a runtime dependency on
-`paper_trail-association_tracking` and keep running the existing tests related
-to association tracking. This arrangement will be maintained for a few years, if
-practical.
+If you would like to track and reify associations you can utilize the following PT plugin: [paper_trail-association_tracking][6]
+
+A little history lesson, discussed as early as 2009, and first implemented in late 2014, association
+tracking was part of PT core until 2018 as an experimental feature and was use at your own risk. The gem now
+maintains a list of known issues and we hope the community can help remove some of them via PR's.
```
Comment (jaredbeck) [inline README.md]: What was your goal with this paragraph? I think I prefer the existing version.

File: README.md [hunk 3]
```diff
@@ -840,17 +840,15 @@ string, please try the [paper_trail-globalid][37] gem.
...
-for production. During that time a steady stream of issues were reported, there
-were not enough volunteers fixing them, and the list of known issues grew. In
-2018, the feature was moved to a separate gem,
-[paper_trail-association_tracking][6].
-
-To avoid breaking changes, `paper_trail` will have a runtime dependency on
-`paper_trail-association_tracking` and keep running the existing tests related
-to association tracking. This arrangement will be maintained for a few years, if
-practical.
+If you would like to track and reify associations you can utilize the following PT plugin: [paper_trail-association_tracking][6]
```
Comment (jaredbeck) [inline README.md]: Good idea to start the section with a simple, introductory sentence and a link.

File: README.md (hunk 2 above)
Comment (westonganger) [reply README.md]: I think the tone has improved

Comment (jaredbeck) [reply README.md]: I agree the existing version sounds a bit negative. I was trying to explain why the gem was extracted, that's why I had that negative-sounding focus on the list of known issues. Maybe we could go with what you have but also direct people to https://github.com/paper-trail-gem/paper_trail/issues/1070 if they want to know why it happened?

Comment (westonganger) [reply README.md]: Sure just write something up that you like. I mainly was concerned about the tone thats all.

**Comment by jaredbeck** [comment]
Closed via 72b3f2c, thanks.

**Pull request closed without merging**